spec/controllers/queries_controller_spec.rb

```diff
@@ -264,27 +264,19 @@ def run_query(id, params = {})
         end
 
         it "should limit the results in JSON response" do
-          begin
-            original_const = DataExplorer::QUERY_RESULT_DEFAULT_LIMIT
-            DataExplorer.send(:remove_const, "QUERY_RESULT_DEFAULT_LIMIT")
-            DataExplorer.const_set("QUERY_RESULT_DEFAULT_LIMIT", 2)
-
-            q = make_query <<~SQL
+          SiteSetting.data_explorer_query_result_limit = 2
+          q = make_query <<~SQL
```

Comment:
Minor but since we''re touching this line, I think it is a good opportunity to improve the variable name.

Reply:
Good point, will change to `query`. Man I dislike 1 letter variables unless they are for a loop or something!